### PR TITLE
Added documentation for properties AccountInfo.*

### PR DESCRIPTION
**Pull request by kenden**
{{AccountInfo.Region}} is referenced in Issue https://github.com/zalando-stups/senza/issues/155
I found the other properties in this PR:
https://github.com/zalando-stups/senza/pull/75

### CODE REVIEW

File: components/senza.rst [hunk 1]
```diff
@@ -307,6 +307,21 @@ You can also combine parameter file and parameters from command line, but you ca
...
+``{{AccountInfo.Region}}`` : the AWS region where the stack is created. Ex: 'eu-central-1'. 
+Note: in many places of a template, `{"Ref" : "AWS::Region"}` can also be used.
+
+``{{AccountInfo.AccountAlias}}`` : the alias name of the AWS account: ex: 'super-team1-account'
+
+``{{AccountInfo.AccountID}}`` : the AWS account id: ex: '353272323354'
+
+``{{AccountInfo.TeamID}}`` : the team ID. Ex: 'super-team1'.
+
+`{{AccountInfo.Domain}}`: the AWS account domain: Ex: super-team1.net
```
Comment (rafaelcaricio) [inline components/senza.rst]: It is missing the second ` here.

Comment (kenden) [inline components/senza.rst]: @rafaelcaricio Thanks, I fixed that.

**Comment by hjacobs** [comment]
Cool, thanks :smile:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**